Source/Scene/Camera.js

```diff
@@ -2299,6 +2315,8 @@ define([
      * @param {Matrix4} [options.endTransform] Transform matrix representing the reference frame the camera will be in when the flight is completed.
      * @param {Boolean} [options.convert=true] When <code>true</code>, the destination is converted to the correct coordinate system for each scene mode. When <code>false</code>, the destination is expected
      *                  to be in the correct coordinate system.
+     * @param {Number} [options.altitude] The maximum altitude at the peak of the flight.
```

Comment:
Throughout Cesium, we call this `height`.  We should do the same here unless we have a compelling reason.